Cargo.toml

```diff
@@ -2,16 +2,15 @@
 name = "rendezvous-server"
 version = "0.1.0"
 edition = "2018"
-
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 anyhow = "1"
 futures = { version = "0.3", default-features = false }
-tokio = { version = "1", features = [ "rt-multi-thread", "time", "macros", "sync", "process", "fs", "net" ] }
+libp2p = { git = "https://github.com/comit-network/rust-libp2p.git", rev = "2e143a01aebbbf318231e7841884f9f34638c4e2", default-features = false, features = [ "rendezvous", "tcp-tokio", "yamux", "mplex", "dns-tokio", "noise", "ping" ] }
```

Comment:
If you wouldn't specify the rev but just the branch, it would be easier to update using `cargo update`.